### ISSUE TITLE
Two Python files?

### ISSUE THREAD

**Issue by Pikamander2** [davidgossage/internal-links#1]
The repo seems to contain two different internal-link-finder.py files, though one is missing its file extension.

Why are there two files and which one is meant to be used?

**Comment by Pikamander2** [comment]
Here's one: https://github.com/davidgossage/internal-links/blob/master/internal-link-finder

The other is in the ZIP file.

**Comment by Pikamander2** [comment]
The only difference seems to be that one of the files has a few extra line breaks.

**Comment by Pikamander2** [comment]
Oh, I guess the ZIP file has an example in it.

**Comment by davidgossage** [comment]
> Oh, I guess the ZIP file has an example in it.

That is correct. Both files work but the zip file has sample CSVs which are needed for the script to work.
